cmd/calyptia/update_pipeline_cluster_objects.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	cloud "github.com/calyptia/api/types"
+)
+
+func newCmdUpdatePipelineClusterObjects(config *config) *cobra.Command {
```

Comment:
Same, this is updating a single cluster object.

```suggestion
func newCmdUpdatePipelineClusterObject(config *config) *cobra.Command {
```